.github/workflows/automerge.yml

```diff
@@ -0,0 +1,48 @@
+name: automerge
+on:
+  pull_request:
+    types:
+      - labeled
+      - unlabeled
+      - synchronize
+      - opened
+      - edited
+      - ready_for_review
+      - reopened
+      - unlocked
+  pull_request_review:
+    types:
+      - submitted
```

Comment:
Need to change this, because every time someone leaves a comment this PR it triggers the automerge GH action: https://github.com/openservicemesh/osm/actions/runs/549416822

Reply:
Seems like there is no better way, so we can keep this as is.